server/views/components/riskWidgets/mappa-widget/template.njk

```diff
@@ -1,5 +1,14 @@
 <div class="mappa-widget">
-    <h3 class="govuk-heading-m"><strong>{{ params.level | default("NO", true) }}</strong> MAPPA</h3>
+  {% if params.status === 'retrieved' %}
+    <h3 class="govuk-heading-m"><strong>{{ params.value.level | default("NO", true) }}</strong> MAPPA</h3>
     <p class="govuk-body-m">Multi-agency public protection arrangements</p>
-    {% if params.level %}<p class="govuk-hint govuk-body-m">Last updated: {{ params.lastUpdated | default("Not known") }}</p>{% endif %}
+    {% if params.value.level %}
+      <p class="govuk-hint govuk-body-m">Last updated: {{ formatDate(params.value.lastUpdated) if params.value.lastUpdated else "Not known" }}</p>
+    {% endif %}
+  {% else %}
+    <h3 class="govuk-heading-m"><strong>UNKNOWN</strong> MAPPA</h3>
+    <p class="govuk-body-m">Multi-agency public protection arrangements</p>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+    <p class="govuk-body-m">Something went wrong. We are unable to show MAPPA information at this time. Try again later.</p>
```

Comment:
I am wondering about our suggestion of "try again" here. I think there is a higher chance that this person hasn't got the necessary data upstream and a lower chance that our request isn't honoured (retry able). If there is no data there's little value in asking the user to retry and invites the question of: "what constitutes a retry, should I start the referral again?", which is something we probably don't want to suggest.

How about we reframe this towards: 

> Something went wrong. We are unable to show or copy MAPPA information at this time. This risk data must be checked manually outside of this service.

My hope is that points the user into continuing their referral on the understanding both they and any assessors will need to go else where for the risk data. 

The same pattern of question applies to our other widgets.

Reply:
Good shout, and I like that wording. Retry here would be "start the application again", given that it's attached to the application at creation time